substrate-node/pallets/pallet-smart-contract/src/cost.rs

```diff
@@ -244,10 +244,15 @@ pub fn calculate_cost_in_tft_from_musd<T: Config>(
     total_cost: u64,
 ) -> Result<u64, DispatchErrorWithPostInfo> {
     let avg_tft_price = pallet_tft_price::AverageTftPrice::<T>::get();
-    ensure!(avg_tft_price > 0, Error::<T>::TFTPriceValueError);
```

Comment:
I think TFTPriceValueError can be removed from error list since it is not used anymore

Reply:
Errors are an Enum, removing them is Unsafe